docs/diagrams/MeetingStatsSequenceDiagram4.puml

```diff
@@ -12,20 +12,20 @@ mainframe **sd** getPreviousMonthlyData
 
 box
```

Comment:
Think you missed out making the return arrow dashed for the self-invocation in this diagram and in the 5th diagram

Reply:
Thanks for the catch, I edited wrongly.